src/components/Span/Span.tsx

```diff
@@ -26,7 +27,9 @@ export const Span = (props: SpanNodeProps) => {
 
   return (
     <div
-      className="flex items-center hover:bg-gray-700 cursor-pointer h-full text-sm"
+      className={`flex items-center hover:bg-gray-700 cursor-pointer h-full text-sm ${
+        props.isSelected ? 'bg-gray-600 hover:bg-gray-700' : ''
```

Comment:
This is a very soft highlight:

<img width="1702" height="779" alt="Image" src="https://github.com/user-attachments/assets/491343e4-f380-4ed5-b12c-0b8987395af4" />

Hard to see. 

Could you instead maybe play with the z-index of the row and have over the overlay?